### ISSUE TITLE
decl. node important boundaries 

### ISSUE THREAD

**Issue by MohammadYounes** [postcss/postcss#966]
Similar to (#598), If a declaration value ends with a comment along with `!important`, that comment will be considered as part of `node.important` rather than being part of `node.raws.value`.

For example:
```css
body {
 font-family:Tahoma/*comment*/!important;
}
```
Produces:
``` js
{ raws:
  { 
     before: '\n',
     between: ': ',
     important: '/*comment*/!important'
  }
}
```

While it is expected to be:

```js
{ raws:
   { 
     before: '\n',
     between: ':  ',
     important: '!important',
     value:{
        raw: 'Tahoma /*comment*/',
        value: 'Tahoma',
     }
   }
}
```
Shouldn't the comment be part of the value ?

Thanks.

**Comment by ai** [comment]
Sure. It is issue. I will fix it in next few weeks. Feel free to ping me on Sunday.

**Comment by ai** [comment]
Done https://github.com/postcss/postcss/commit/cb45067009fb8e7ecc1c64372a4100cf61192ef0

But I need to move this test to `postcss-parser-tests`.

**Comment by ai** [comment]
I moved test to `postcss-parser-tests` https://github.com/postcss/postcss-parser-tests/commit/bdeac1138ec1600a71fb0781287e795d6c33b857 , but I will release it tomorrow. Too tired today.

**Comment by MohammadYounes** [comment]
Awesome! What about #598 ?

**Comment by ai** [comment]
Seems like i forgot about it :). But I spend all PostCSS time for this week. Could we fix #598 on next week? Just remind me in Friday.

**Comment by MohammadYounes** [comment]
Sure, I hope I don't forget ;)

Thanks!

**Comment by ai** [comment]
Released in 5.2.13

**Comment by MohammadYounes** [comment]
@ai  in case the comment was placed after `!important` it will produce:
```js
{ raws:
  { 
     before: '\n',
     between: ': ',
     important: '!important/*comment*/'
  }
}
```

https://runkit.com/5834c46bd8fb6400141ac375/58c84fe87474fc0014f16da2
